src/vivarium/framework/results/__init__.py

```diff
@@ -0,0 +1,4 @@
+from vivarium.framework.results.manager import (
+    ResultsManager,
```

Comment:
Why does the `ResultsManager` need to be exposed here?

Reply:
To match the interface of other system packages that are framework level modules.